ha_lineairdb.h

```diff
@@ -67,6 +67,29 @@ class LineairDB_share : public Handler_share {
   std::unique_ptr<LineairDB::Database> lineairdb_;
 };
 
+class Mysql_lineairdb_translator
+{
+public:
+  std::string nullFlags_;
+  std::string mysqlField_;
+private:
+  std::string lineairdbField_;
+
+  size_t read_header(const std::byte* const field, std::string& valBuf, size_t offset);
+  void encode_db_header(const size_t input);
+public:
```

Comment:
I prefer public methods/fields to precede private methods/fields. 
This way makes code easier to read, and I can notice that it would be unnecessary to read after the `private:`.

Reply:
Understood, thank you👍